src/test/java/cappy/parser/ParsedInputTest.java

```diff
@@ -0,0 +1,78 @@
+package cappy.parser;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import cappy.command.CommandType;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class ParsedInputTest {
+    @Test
+    public void numberOfPositionalArguments() {
```

Comment:
Understand that it is already labelled with "@Test", but maybe consider adding Test to the function name

e.g.

```java
  @Test
  public void anotherDummyTest(){
      assertEquals(4, 4);
  }
```